pom.xml

```diff
@@ -53,13 +53,29 @@
         <dependency>
             <groupId>com.datastax.oss</groupId>
             <artifactId>java-driver-core</artifactId>
-            <version>4.4.0</version>
+            <version>4.13.0</version>
         </dependency>
         <dependency>
             <groupId>javax.validation</groupId>
             <artifactId>validation-api</artifactId>
             <version>2.0.0.Final</version>
         </dependency>
+	<!-- Override from underlying datastax dependency -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.13.0</version>
```

Comment:
Can we use a property so that Jackson dependencies are synced? My understanding is that they release versions of submodules in lockstep.